ot/gromov/_utils.py

```diff
@@ -522,3 +524,254 @@ def h2(b):
     hC2 = h2(C2)
     fC2t = f2(C2).T
     return constC, hC1, hC2, fC2t
+
+
+############################################################################
+# Methods related to fused unbalanced GW and unbalanced Co-Optimal Transport.
+############################################################################
+
+# Calculate Div(pi, a \otimes b) for Div = KL and squared L2.
+# Calculate KL(pi, a \otimes b) using
+# the marginal distributions pi1, pi2 of pi
+def approx_shortcut_kl(pi, pi1, pi2, a, b, nx=None):
```

Comment:
suggestion for a better name and API:
approx_shortcut_kl(pi, pi1, pi2, a, b, nx=None) -> kl_div_to_product(pi, a, b, pi1=None, pi2=None, nx=None)
Plus set doc environment (:math:, parameters etc)